scripts/i18n/weblate-setup.sh

```
@@ -31,7 +31,7 @@ FILE_BASE="frontend/src/i18n"
 SOURCE_LANG="en"
 
 # All languages the project supports
-LANGUAGES="en es de fr hr it ja nl pl pt ru sv zh hu sl sk uk id da"
+LANGUAGES="en es de fr hr it ja nl pl pt ru sv zh hu sl sk uk id da ko cs"
```

Shellcheck (0.11.0)

[warning] LANGUAGES appears unused. Verify use (or export if used externally).

(SC2034)